docs/concepts/index.md

```diff
@@ -5,7 +5,12 @@ The Concepts section of the Kubernetes documentation is a work in progress.
 
 #### Object Metadata
 
-[Annotations](/docs/concepts/object-metadata/annotations/)
+
+* [Annotations](/docs/concepts/object-metadata/annotations/)
+
+#### Controllers
+* [StatefulSets](/docs/concepts/controllers/statefulsets/)
```

Comment:
There is an "abstractions" directory inside concepts that I'd like "controllers" to be nested under.